File: Praktikum4/ProgramPython/PRAK404-2010817210023-MuhammadAuliaAkbar.py
while True:
    print("Pilih Program: ")
    print("1. Penjumalahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    masukanInput = int(input("Masukan Pilihan: "))

    if masukanInput == 5:
        print("Terimakasih, telah menggunakan kalkulator Muhammad Aulia Akbar")
        break

    if masukanInput == 1:
        print("Masukan Nilai Pertama ", end="")
        nilai_pertama = float(input())
        print("Masukan Nilai Kedua ", end="")
        nilai_kedua = float(input())

        hasil = nilai_pertama + nilai_kedua 
        print(f"Hasil Penjumlahan antara {nilai_pertama} dan {nilai_kedua} adalah {hasil}")

    elif masukanInput == 2:
        print("Masukan Nilai Pertama ", end="")
        nilai_pertama = float(input())
        print("Masukan Nilai Kedua ", end="")
        nilai_kedua = float(input())

        hasil = nilai_pertama - nilai_kedua 
        print(f"Hasil Penjumlahan antara {nilai_pertama} dan {nilai_kedua} adalah {hasil}")

    elif masukanInput == 3:
        print("Masukan Nilai Pertama ", end="")
        nilai_pertama = float(input())
        print("Masukan Nilai Kedua ", end="")
        nilai_kedua = float(input())

        hasil = nilai_pertama * nilai_kedua 
        print(f"Hasil Penjumlahan antara {nilai_pertama} dan {nilai_kedua} adalah {hasil}")

    elif masukanInput == 4:
        print("Masukan Nilai Pertama ", end="")
        nilai_pertama = float(input())
        print("Masukan Nilai Kedua ", end="")
        nilai_kedua = float(input())

        hasil = nilai_pertama / nilai_kedua 
        print(f"Hasil Penjumlahan antara {nilai_pertama} dan {nilai_kedua} adalah {hasil}")
    
    else:
        print("Input anda salah, silahkan coba lagi")

File: Praktikum4/ProgramC/PRAK404-2010817210023-MuhammadAuliaAkbar.c
#include <stdio.h>
int main()
{
    int pilInputan;
    float hasil, nPertama, nKedua;
    do
    {
        printf("\n\n\nPilih Program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukan Pilihan: ");
        scanf("%d", &pilInputan);

        switch (pilInputan)
        {
        case 1:
            printf("Masukan nilai pertama: ");
            scanf("%f", &nPertama);
            printf("Masukan nilai kedua: ");
            scanf("%f", &nKedua);
            hasil = nPertama + nKedua;
            printf("Hasil Penjumlahan antara %.0f dengan %.0f adalah %.0f", nPertama, nKedua, hasil);
            break;
        case 2:
            printf("Masukan nilai pertama: ");
            scanf("%f", &nPertama);
            printf("Masukan nilai kedua: ");
            scanf("%f", &nKedua);
            hasil = nPertama - nKedua;
            printf("Hasil Pengurangan antara %.0f dengan %.0f adalah %.0f", nPertama, nKedua, hasil);
            break;
        case 3:
            printf("Masukan nilai pertama: ");
            scanf("%f", &nPertama);
            printf("Masukan nilai kedua: ");
            scanf("%f", &nKedua);
            hasil = nPertama * nKedua;
            printf("Hasil Perkalian antara %.0f dengan %.0f adalah %.0f", nPertama, nKedua, hasil);
            break;
        case 4:
            printf("Masukan nilai pertama: ");
            scanf("%f", &nPertama);
            printf("Masukan nilai kedua: ");
            scanf("%f", &nKedua);
            hasil = nPertama / nKedua;
            printf("Hasil Pembagian antara %.0f dengan %.0f adalah %.2f", nPertama, nKedua, hasil);
            break;
        case 5:
            printf("Program telah berakhir...\n");
            printf("Terimakasih, telah menggunakan kalkulator Muhammad Aulia Akbar");
            break;
        default:
            printf("Input anda salah, silahkan coba lagi");
            break;
        }
    } while (pilInputan != 5);

    return 0;
}